### DISCUSSION TITLE
Select2 in Editable Columns

### DISCUSSION THREAD

**Discussion by julienmru** [Laravel-Backpack/community-forum#1249] [Ideas]
I've been using Editable Columns and I love it. However, when we have lots of options in a Editable Select, we don't find it very user-friendly and would rather switch to Select2. Ideally both from array and from an AJAX endpoint. For the AJAX endpoint, the primary key of the record should be passed as a parameter. AJAX endpoint is low-priority for us ;)

[a comment by an automated account is not shown]

**Comment by julienmru** [comment]
Thanks to [@backpack-ai-bot](https://github.com/apps/backpack-ai-bot) for pointing out https://github.com/Laravel-Backpack/community-forum/discussions/1046 😄 

@jcastroa87 Do you have more progress? Or an ETA?

**Reply by karandatwani92** [reply]
Hey @julienmru 

Imagine having multiple `select2` for multiple `columns` dynamically set up inside a dynamic table rendered by datatables. It could be a mess. Thus we may postpone or discontinue the progress that way. This is what I remember from our past team discussion.

@jcastroa87, please correct me if I need to be corrected on this.

**Reply by jcastroa87** [reply]
Hello, yes, we will not make any progress with select2 in editable column, it is labeled as Won't and probably will be close soon.

Cheers.
